test/forms/declaration/WarehouseIdentificationSpec.scala

```diff
@@ -27,7 +29,9 @@ class WarehouseIdentificationSpec extends WordSpec with MustMatchers {
       val warehouseIdentification = correctWarehouseIdentification
       val expectedMetadataProperties: Map[String, String] = Map(
         "declaration.goodsShipment.warehouse.id" -> warehouseId,
-        "declaration.goodsShipment.warehouse.typeCode" -> warehouseTypeCode
+        "declaration.goodsShipment.warehouse.typeCode" -> warehouseTypeCode,
```

Comment:
It would be nice to test instead the values assigned not by the mapping, but actually by the model. An example of this test can be seen here https://github.com/hmrc/customs-declare-exports-frontend/blob/master/test/forms/declaration/ConsigneeDetailsSpec.scala#L28

Reply:
Yeah, you're right, but as I discussed with @raghu1978, because he's working around transport information page, he'll do all stuff with this mapping. Just like we discussed, this PR is the first PR for this task, there will be another one :)